book/src/list-functions.md

```diff
@@ -74,6 +74,7 @@ fn sech(x: Scalar) -> Scalar
 fn asech(x: Scalar) -> Scalar
 fn csch(x: Scalar) -> Scalar
 fn acsch(x: Scalar) -> Scalar
+fn quadratic_equation<A2, B2>(a: A2, b: B2, c: B2²/A2) -> String
```

Comment:
Can we please move this to a new algebra section at the bottom of this page?

Reply:
Sure, let me know if [5c7f33b](https://github.com/sharkdp/numbat/pull/398/commits/5c7f33b2b2a52b18b72d426087ca11b7cfd07436) isn't what you wanted.